webpack.config.js

```diff
@@ -81,8 +86,19 @@ module.exports = (env) => {
             org: 'neo4j-inc',
             project: 'neodash',
           }),
+          new webpack.DefinePlugin({
+            process: { env: {} },
+          }),
         ]
-      : [new ReactRefreshWebpackPlugin(), ...(circularValidation ? [circularPlugin] : [])],
+      : [
+          new webpack.HotModuleReplacementPlugin(),
```

Comment:
Do we need both HotModuleReplacementPlugin and ReactRefreshWebpackPlugin?

Reply:
not sure, I can look at what is in the main branch and see if that works

Reply:
I've removed HotModuleReplacementPlugin and the code refresh still works